docs/BR.md

```diff
@@ -380,6 +382,10 @@ The Definitions found in the CA/Browser Forum's Network and Certificate System S
 
 **Legal Entity**: An association, corporation, partnership, proprietorship, trust, government entity or other entity with legal standing in a country's legal system.
 
+**Multi-Perspective Issuance Corroboration**: A process by which the determinations made during domain validation and CAA checking by the Primary Network Perspective are corroborated from other Network Perspectives before Certificate issuance. 
+
+**Network Perspective**: Related to Multi-Perspective Issuance Corroboration. A system for sending outbound Internet traffic associated with a domain control validation method and CAA check. The location of a Network Perspective is determined by the point where unencapsulated outbound Internet traffic is first handed off to the network infrastructure providing Internet connectivity to that perspective.
```

Comment:
```suggestion
**Network Perspective**: Related to Multi-Perspective Issuance Corroboration. A system for sending outbound Internet traffic associated with a domain control validation method and/or CAA check. The location of a Network Perspective is determined by the point where unencapsulated outbound Internet traffic is first handed off to the network infrastructure providing Internet connectivity to that perspective.
```

Reply:
This appears to add flexibility to how CA Owners implement MPIC, and we think that’s a good thing. Using specific Network Perspectives for either DCV or CAA checks could create opportunities for operational efficiencies, which we had no intention of limiting.

We pre-staged this change in https://github.com/ChristopherRC/servercert/commit/07f9c3bac410a8af53ef60755b2788e67f7ab556 in preparation for a subsequent round of discussion.

Reply:
This can be read as "well, we did MP for CAA, so we don't need to do it for DV". Is that the intention here?

Reply:
@aaomidi - the definition is intended to serve the most general use case and not inadvertently limit CA Owner flexibility re: implementation. 

For example, if we assume I need 5 "successful" DCV/CAA lookups, I should be able to:
- perform the lookups from 5 Network Perspectives (each perspective evaluates both DCV and CAA), or
- perform the lookups from 10 Network Perspectives (5 perspectives evaluate DCV, 5 others CAA).

Regardless of the above, the requirements in Section 3.2.2.9 make expectations clear (especially those [staged](https://github.com/ChristopherRC/servercert/blob/SC-067-Version-2/docs/BR.md#3229-multi-perspective-issuance-corroboration) for the next round of discussion) that both DCV and CAA need to be considered.